[CI] ReindexClientYamlTestSuiteIT test {yaml=reindex/10_basic/Response format for updated} failing

**Build Scans:**
- [elasticsearch-intake #38841 / part1](https://gradle-enterprise.elastic.co/s/zbaf7n4h4sj3s)

**Reproduction Line:**
```
./gradlew ":modules:reindex:yamlRestTest" --tests "org.elasticsearch.index.reindex.ReindexClientYamlTestSuiteIT.test {yaml=reindex/10_basic/Response format for updated}" -Dtests.seed=2EB38C13FA15E46B -Dtests.locale=oc-ES -Dtests.timezone=Etc/GMT-9 -Druntime.java=26
```

**Applicable branches:**
9.4

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.index.reindex.ReindexClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Byaml%3Dreindex%2F10_basic%2FResponse%20format%20for%20updated%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: java.lang.AssertionError: 
Expected: <0>
     but: was <1>
```

**Issue Reasons:**
- [9.4] 1 consecutive failures in test test {yaml=reindex/10_basic/Response format for updated}
- [9.4] 1 consecutive failures in step part1
- [9.4] 1 consecutive failures in pipeline elasticsearch-intake

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -288,9 +288,6 @@ tests:
 - class: org.elasticsearch.test.rest.ClientYamlTestSuiteIT
   method: test {yaml=cluster.info/20_info_http/Cluster HTTP Info}
   issue: https://github.com/elastic/elasticsearch/issues/147753
-- class: org.elasticsearch.index.reindex.ReindexClientYamlTestSuiteIT
-  method: test {yaml=reindex/10_basic/Response format for updated}
-  issue: https://github.com/elastic/elasticsearch/issues/148055
 - class: org.elasticsearch.xpack.esql.qa.mixed.EsqlClientYamlIT
   method: test {p0=esql/50_index_patterns/_index LIKE with overly complex pattern}
   issue: https://github.com/elastic/elasticsearch/issues/148134

